src/pages/Invoice/CreateInvoice.js

```diff
@@ -9,10 +9,14 @@ import { useNotification } from "notification";
 import { useForm } from "customHooks/useForm";
 
 const CreateInvoice = () => {
-    const [modalStatus, setModalStatus] = useState(false);
-    const [customersInfo, setCustomersInfo] = useState([]);
-    const [itemInfo, setItemInfo] = useState([]);
-    const [fields, handleFieldChange, setState] = useForm({
+    const [modalState, setModalState] = useState({
+        status: false,
+        items: [],
+        customers: [],
```

Comment:
Please, take a pull from the main branch, the str for customers have been changed to object